proto/src/einride/iam/example/v1/freight_service.proto

```diff
@@ -346,6 +351,17 @@ service FreightService {
     };
   }
 
+  option (einride.iam.v1.operations_permissions) = {
+    operation_permissions {
+      operation {type: "iam-example.einride.tech/ShipperOperation"}
```

Comment:
I always thought this was some kind of JSON syntax, but this sure doesn't look like it. I guess it's some special syntax then?

Reply:
Yes, it's actually proto text syntax: https://pkg.go.dev/google.golang.org/protobuf/encoding/prototext